pulp_container/app/migrations/0019_DATA_distribution_model_swap.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 2.2.19 on 2021-03-19 17:42
+
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+
+
+def migrate_data_from_old_model_to_new_model_up(apps, schema_editor):
+    """ Move objects from ContainerDistribution to NewContainerDistribution."""
+    AnsibleDistribution = apps.get_model('container', 'ContainerDistribution')
```

Comment:
s/AnsibleDistribution/ContainerDistribution/

Reply:
How did I even miss that :/